redis-hbo-provider/src/main/java/com/facebook/presto/statistic/RedisProviderPlugin.java

```diff
@@ -49,6 +48,12 @@ public RedisProviderPlugin(String configPath)
         this.propertyMap = propertyMap;
```

Comment:
call the below constructor instead of setting it directly

Reply:
` Call to 'this()' must be first statement in constructor body` Java doesn't allow that

Reply:
oh yup. wasn't thinking about all the stuff happening above